docs/source/compression/compression_config_list.rst

```diff
@@ -1,11 +1,190 @@
 Compression Config Specification
 ================================
 
+Common Keys in Config
+---------------------
+
+op_types
+^^^^^^^^
+
+A list of type names of classes that inherit from ``torch.nn.Module``.
+Only module types in this list can be selected to be compressed.
+If this key is not set, all module types can be selected.
```

Comment:
"can be" -> "are"? do we have default list for op types, or we really support all module types?

Reply:
It's `can be`, if `op_names` or `op_names_re` is not set, it becomes `are`.

add `If neither ``op_names`` or ``op_names_re`` are set, all modules satisfied the ``op_types`` are selected.`

We don't have default value for `op_types`, but by default we support `'Linear', 'Embedding', 'Conv1d', 'Conv2d', 'Conv3d', 'ConvTranspose1d', 'ConvTranspose2d', 'ConvTranspose3d', 'BatchNorm1d', 'BatchNorm2d', 'BatchNorm3d'` for pruning target setting, `'Linear', 'Embedding', 'Conv1d', 'Conv2d', 'Conv3d', 'ConvTranspose1d', 'ConvTranspose2d', 'ConvTranspose3d', 'ReLU', 'ReLU6'` for quantization target setting, and all types for distillation target setting.
And we support all module types by user customization.

Reply:
should we add these information here?